engine/apps/api/views/user.py

```diff
@@ -841,6 +842,25 @@ def export_token(self, request, pk) -> Response:
             return Response(status=status.HTTP_204_NO_CONTENT)
         return Response(status=status.HTTP_405_METHOD_NOT_ALLOWED)
 
+    @action(methods=["get"], detail=False)
+    def filters(self, request):
+        filter_name = request.query_params.get("search", None)
+        api_root = "/api/internal/v1/"
+
+        filter_options = [
+            {
+                "name": "team",
+                "type": "team_select",
+                "href": api_root + "teams/",
+                "global": True,
+            },
+        ]
+
+        if filter_name is not None:
+            filter_options = list(filter(lambda f: filter_name in f["name"], filter_options))
```

Comment:
does the frontend use this search functionality? if not, I think it might make sense to remove this

Reply:
yes, it does

Reply:
I can't find any usages in the frontend code 🤔 Am I missing something?